src/ansys/fluent/core/post_objects/singleton_meta.py

```diff
@@ -0,0 +1,20 @@
+"""Provides a module for Metaclasses."""
+
+from abc import ABCMeta
+
+
+class SingletonMeta(type):
+    """Meta class for singleton type."""
```

Comment:
```suggestion
    """Provides the metaclass for the singleton type."""
```
Google search shows it as one word, not capitalized.